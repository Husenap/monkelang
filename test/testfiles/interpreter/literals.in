monke say 1
monke say 2
monke say true
monke say false
monke say "Hello, World!"
monke say "This\tis a \"string\"\nincluding some escape sequences"
monke say "       World!\rHello,"
