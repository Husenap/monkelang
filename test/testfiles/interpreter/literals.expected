1
2
true
false
Hello, World!
This	is a "string"
including some escape sequences
       World!Hello,
